src/Compilers/Core/Portable/Operations/OperationFactory.cs

```diff
@@ -15,5 +12,12 @@ public static IInvalidOperation CreateInvalidOperation(SemanticModel semanticMod
         {
             return new InvalidOperation(children, semanticModel, syntax, type: null, constantValue: null, isImplicit: isImplicit);
         }
+
+        public static readonly IConvertibleConversion IdentityConversion = new IdentityConvertibleConversion();
```

Comment:
>IdentityConversion [](start = 54, length = 18)

Is this shared instance used in `IOperation` trees? If so, how do we handle the parent pointer?